integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy

```diff
@@ -149,157 +144,423 @@ class AuthorizationServerIT extends ITSupport {
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://example"])
+        )
+        registerForCleanup(createdAuthorizationServer)
+        assertThat(createdAuthorizationServer, notNullValue())
+        assertThat(createdAuthorizationServer.getStatus(), equalTo(AuthorizationServer.StatusEnum.ACTIVE))
+
+        createdAuthorizationServer.deactivate()
+
+        // delete operation may not effect immediately in the backend
+        sleep(getTestOperationDelay())
+
+        AuthorizationServer retrievedAuthorizationServer = client.getAuthorizationServer(createdAuthorizationServer.getId())
+        assertThat(retrievedAuthorizationServer, notNullValue())
+        assertThat(retrievedAuthorizationServer.getId(), equalTo(createdAuthorizationServer.getId()))
+        assertThat(retrievedAuthorizationServer.getStatus(), equalTo(AuthorizationServer.StatusEnum.INACTIVE))
+    }
+
+    @Test
+    void activateAuthorizationServerTest() {
+        String name = "java-sdk-it-" + UUID.randomUUID().toString()
+
+        AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://example"])
+        )
+        registerForCleanup(createdAuthorizationServer)
+        assertThat(createdAuthorizationServer, notNullValue())
+        assertThat(createdAuthorizationServer.getStatus(), equalTo(AuthorizationServer.StatusEnum.ACTIVE))
+
+        createdAuthorizationServer.deactivate()
+
+        // delete operation may not effect immediately in the backend
+        sleep(getTestOperationDelay())
+
+        AuthorizationServer retrievedAuthorizationServer = client.getAuthorizationServer(createdAuthorizationServer.getId())
+        assertThat(retrievedAuthorizationServer, notNullValue())
+        assertThat(retrievedAuthorizationServer.getId(), equalTo(createdAuthorizationServer.getId()))
+        assertThat(retrievedAuthorizationServer.getStatus(), equalTo(AuthorizationServer.StatusEnum.INACTIVE))
+
+        createdAuthorizationServer.activate()
+
+        // delete operation may not effect immediately in the backend
+        sleep(getTestOperationDelay())
+
+        retrievedAuthorizationServer = client.getAuthorizationServer(createdAuthorizationServer.getId())
+        assertThat(retrievedAuthorizationServer, notNullValue())
+        assertThat(retrievedAuthorizationServer.getId(), equalTo(createdAuthorizationServer.getId()))
+        assertThat(retrievedAuthorizationServer.getStatus(), equalTo(AuthorizationServer.StatusEnum.ACTIVE))
+    }
+
     // Policy operations
 
     @Test
     void listAuthorizationServerPoliciesTest() {
+        AuthorizationServerList authorizationServerList = client.listAuthorizationServers()
```

Comment:
Do a 'createAuthorizationServer()` before you do `listAuthorizationServers()`.

Reply:
This test-method designed to verify `"GET - /api/v1/authorizationServers"` and do not requeire AuthorizationServer creation.

Test for `"POST - /api/v1/authorizationServers"` createAuthorizationServer() implemented in [here](https://github.com/okta/okta-sdk-java/blob/c4fb5f69a98619a5071f6ec711ca822baa07c3fc/integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy#L50-L56)

Reply:
"Within the scope" of this test, you should do the below before doing a `client.listAuthorizationServers()`.

```
AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
            client.instantiate(AuthorizationServer)
                .setName(name)
                .setDescription("Test Authorization Server")
                .setAudiences(["api://example"])
        )
        registerForCleanup(createdAuthorizationServer)
```

Reason is that all these tests are run in parallel and you cant guarantee the order in which they would be executed in test env. Also, the create that is done at https://github.com/okta/okta-sdk-java/blob/c4fb5f69a98619a5071f6ec711ca822baa07c3fc/integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy#L50-L56 will be cleaned up (deleted) once the test method exits (`registerForCleanup(createdAuthorizationServer)`) cleans up the created resource).